src/components/actions/processTrigger.ts

```diff
@@ -106,7 +106,10 @@ export async function processTrigger(payload: TriggerPayload): Promise<ActionRes
 
       // Debug helper console.log to inspect action outputs:
       if (showActionOutcomeLog) console.log('outputCumulative:', outputCumulative)
-      if (result.status === ActionQueueStatus.Fail) console.log(result.error_log)
+      if (result.status === ActionQueueStatus.Fail) {
+        console.log(result.error_log)
+        actionFailed = action.action_code
+      }
```

Comment:
It was behaving as expected when the action actually threw an error, but most of the time now we're catching them in the action and returning the error status in the `result`, but this wasn't being acted on when the status is "FAIL".